### PR TITLE
Implement do-while to while loop transform

### PR DESCRIPTION
**Pull request by joaonmatos**

### CODE REVIEW

**Review by joaobispo — APPROVED** [review]

File: ClavaLaraApi/src-lara-clava/clava/clava/code/DoToWhileStmt.js
Comment (joaobispo) [inline ClavaLaraApi/src-lara-clava/clava/clava/code/DoToWhileStmt.js:7]: Is this copy necessary? If you just do `ClavaJoinPoints.scope($doStmt.scopeNodes) don't the nodes get automatically copied?

**Comment by joaonmatos** [comment]
Current dependencies on/for this PR:
* master
  * **PR #73** <a href="https://app.graphite.dev/github/pr/specs-feup/clava/73" target="_blank">Graphite</a> 
    * **PR #72** <a href="https://app.graphite.dev/github/pr/specs-feup/clava/72" target="_blank">Graphite</a>  👈
      * **PR #76** <a href="https://app.graphite.dev/github/pr/specs-feup/clava/76" target="_blank">Graphite</a> 
        * **PR #74** <a href="https://app.graphite.dev/github/pr/specs-feup/clava/74" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/specs-feup/clava/72?utm_source=stack-comment).

**Comment by joaonmatos** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/specs-feup/clava/72?utm_source=stack-comment).


Job Reference: rvWNJd05cwCkJ6Hklrdh

**Pull request merged**
